examples/dll-app-and-vendor/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var fs = require("fs");
+
+if(!fs.existsSync("./js")) {
+	// build the vendor first,
+	// since automated integration tests
+	// seem to run the webpack config not the build.js
+	require("./build");
```

Comment:
@sokra I will update them as needed, but It would be good to discuss the best way to do that to handle the tests, but also make the examples as easy to read and understandable as possible.